src/containers/auth/forgotPassword/index.jsx

```diff
@@ -0,0 +1,162 @@
+/**
+ * TODO: import prop-types package
+ */
+// import PropTypes from 'prop-types'
+import React, { PureComponent } from 'react';
+import Modal from 'react-modal';
+import { FaTimes } from 'react-icons/fa';
+import { IoIosCheckmarkCircle } from 'react-icons/io';
+
+import styles from './styles.module.scss';
+import { forgotPasswordRequest } from '../../../services/auth';
+import validateAuth from '../../../validation/auth';
+import Spinner from '../../../components/spinner/index';
+
+Modal.setAppElement('#root');
+
+class ForgotPassword extends PureComponent {
+  state = {
+    status: null,
+    data: null,
+    email: null,
+    message: null,
+    pending: false,
+  };
+
+  refInput = React.createRef();
+
+  send = async email => {
```

Comment:
Why this function is async?

Reply:
Because of, inner functions are async and I need to wait for the response from the Server.